x/fswap/simulation/simap.go

```diff
@@ -0,0 +1,15 @@
+package simulation
+
+import (
+	sdk "github.com/Finschia/finschia-sdk/types"
+	simtypes "github.com/Finschia/finschia-sdk/types/simulation"
+)
+
+// FindAccount find a specific address from an account list
+func FindAccount(accs []simtypes.Account, address string) (simtypes.Account, bool) {
+	creator, err := sdk.AccAddressFromBech32(address)
+	if err != nil {
+		panic(err)
+	}
+	return simtypes.FindAccount(accs, creator)
+}
```

Comment:
What is this function for? Is it part of initial?

Reply:
No, this is auto generated from ignite. I found in da $ zkauth, this file left, so I also left it. But I think we will remove it during development.

Reply:
Let's remove those things that we find unnecessary during development